src/incubator-tooltip.html

```diff
@@ -129,7 +134,10 @@
           }
           if (this._targetElement && !this.isAttached) {
             this.isAttached = true;
-            Polymer.dom(this._targetElement).appendChild(this);
+            if (['input', 'img'].indexOf(this._targetElement.tagName.toLowerCase()) === -1) {
```

Comment:
please extract the condition to a boolean variable to improve readability. something like `targetIsEmptyTag`, according to this 
https://www.w3schools.com/html/html_elements.asp